### PR TITLE
Credit section in time line event

### PR DESCRIPTION
**Pull request by busy-john**
The credit section is able to show multiple credit items, with or without link. Data is going to be filled in another PR.

Without link
Screenshot 2019-07-07 at 02 47 55

With link
Screenshot 2019-07-07 at 03 23 23

With link and there's a button on top of credit section
Screenshot 2019-07-07 at 03 23 56

### CODE REVIEW

File: core/src/templates/timeline-page.js [hunk 1]
```diff
@@ -24,17 +25,71 @@ const StyledDesc = styled(Typography)`
...
+            </ListItem>
+          )
+        })
+      }
+    </List>
+  )
+}
+
+function EventFooter({ event }) {
+  const hasCredit = (event.extra.creditText && event.extra.credits)
```
Comment (mingchuno) [inline core/src/templates/timeline-page.js]: have you run `yarn lint` yet? seems we can skip the `()`

File: core/src/templates/timeline-page.js [hunk 2]
```diff
@@ -24,17 +25,71 @@ const StyledDesc = styled(Typography)`
...
+  )
+}
+
+function EventCreditList({ title, credits }) {
+  const validCredits = credits.filter(c => !!c.name)
+  return (
+    <List subheader={<EventCreditListSubheader title={title}/>} dense>
+      {
+        validCredits.map((c, idx) => {
+          if (c.link == null || c.link.length == 0) {
```
Comment (mingchuno) [inline core/src/templates/timeline-page.js]: since `c.link` should be string, so we can
```suggestion
          if (!c.link) {
```

Comment (busy-john) [reply core/src/templates/timeline-page.js]: Fixed. Please check

File: core/src/templates/timeline-page.js (hunk 1 above)
Comment (busy-john) [reply core/src/templates/timeline-page.js]: Fixed, please check

**Pull request merged**